balloon-hash/src/error.rs

```diff
@@ -3,13 +3,14 @@
 use core::fmt;
 
 #[cfg(feature = "password-hash")]
-use password_hash::errors::InvalidValue;
+use ::{core::cmp::Ordering, password_hash::errors::InvalidValue};
```

Comment:
The `::` here is unnecessary.

```suggestion
use {core::cmp::Ordering, password_hash::errors::InvalidValue};
```

Reply:
Didn't know that :astonished:.
Done.